src/library/scala/collection/immutable/Vector.scala

```diff
@@ -466,24 +466,24 @@ override def companion: GenericCompanion[Vector] = Vector
       display5 = copyRange(display5, oldLeft, newLeft)
   }
 
-  private def zeroLeft(array: Array[AnyRef], index: Int): Unit = {
+  private[this] def zeroLeft(array: Array[AnyRef], index: Int): Unit = {
```

Comment:
"it" meaning having the method be final, private[this], or just private like it used to be.  In any case the effects are small in my hands (~5%).